### DISCUSSION TITLE
Using OCAP on a LAN configuration

### DISCUSSION THREAD

**Discussion by GuyMoradDCA** [OCAP2/OCAP#80] [Q&A]
Hey, I've got a sort of LAN party configuration with 20 or so PCs, they all have an updated copy of ARMA 3 but they're offline, we play on them over LAN, is there a good way to use OCAP without having an actual public internet connection?

**Comment by indig0fox** [comment]
As long as the webserver is running on your LAN too, and therefore is the same machine or accessible via IP from the machine running the A3 server, the A3 server will still upload the recording file as long as it's properly configured. Playback is then available from anywhere that can reach (default) port 5000 of the webserver machine.

Configure your webserver config JSON with 0.0.0.0 or it's local IP address as the interface. This is done via setting.json in the web component's root, which the binary reads to determine what to open.


Configure your A3 addon upload destination settings to point to the webserver by updating the URL in see OcapReplaySaver2.cfg.json to the LAN IP of your webserver.

For 1.1.0
https://ocap2.notion.site/Setup-and-Configuration-4a00030a68f84c5895e799629f9dac4e?pvs=4

For 2.0
https://ocap2.notion.site/OCAP2-Addon-CBA-Rework-2-0-80104797eb6443efb59abf49edbc8d31?pvs=4
